### PR TITLE
Wmel update

### PR DESCRIPTION
**Pull request by pcruzparri**
## Changes

please describe your changes here :smiley:

Added transfer arrow customization. Added commenting. Tested plot customization. 

## Checklist

- [ ] added tests, if applicable
- [x] updated documentation, if applicable
- [ ] updated CHANGELOG.md
- [x] tests pass

### CODE REVIEW

File: WrightTools/diagrams/WMEL.py [hunk 1]
```diff
@@ -432,3 +460,193 @@ def plot(self, save_path=None, close=False, bbox_inches="tight", pad_inches=1):
...
+            plot.axis("off")
+            if ind % 2 == 0:
+                if titles != [None]:
+                    subtitles = []
+                    for t in titles:
+                        subtitles.append(t)
+                        subtitles.append("")
+                elif titles == [None]:
+                    subtitles = [""] * (2 * number_of_interactions)
+                print(subtitles)
```
Comment (ksunden) [inline WrightTools/diagrams/WMEL.py:562]: extraneous print

File: WrightTools/diagrams/WMEL.py [hunk 2]
```diff
@@ -432,3 +460,193 @@ def plot(self, save_path=None, close=False, bbox_inches="tight", pad_inches=1):
...
+                        title=subtitles[ind],
+                        title_font_size=title_font_size,
+                        state_names=state_names,
+                        virtual=virtual,
+                        state_font_size=state_font_size,
+                        state_text_buffer=state_text_buffer,
+                        label_side=label_side,
+                    )  # Uses the Subplot class to make the each process diagram
+
+                if ind == len(subtitles) - 2 and label_side == "right":
```
Comment (ksunden) [inline WrightTools/diagrams/WMEL.py:577]: elif to avoid falling through to the else condition for the first in this if/elif chain

Comment (ksunden) [reply WrightTools/diagrams/WMEL.py:577]: This is what is being indicated by the bot, though it's a little hard to decipher and I was at first like "nah, bot, you got that wrong", but it is actually correct

**Comment by untzag** [comment]
I just want to say thanks for the contribution @pcruzparri

**Comment by ksunden** [comment]
Looks good to me in imploementation, I'd perhaps like to see an example added to the gallery showing off the new feature/serving as some form of reference for it's usage:

See https://github.com/wright-group/WrightTools/blob/master/examples/wmels_trive_population_transfer.py for an example of the existing wmel artist gallery entry (just put a script that uses this code in that folder)

I'll merge this for now though, that can be a separate PR

**Review by ksunden — APPROVED** [review]

**Pull request merged**